.pre-commit-config.yaml

```diff
@@ -57,6 +57,9 @@ repos:
             "-rn",
             "-sn",
             "--rcfile=pylintrc",
+            "--fail-under=10",
+            "--evaluation='10.0 - ((float(5 * error + warning + refactor + convention +
+            info) / statement) * 10)'",
```

Comment:
Why not use `--fail-on=useless-suppression` instead?
Maybe `--fail-on=I` works 🤷🏻‍♂️ haven't tested it.

Reply:
`--fail-on` works, forgot that was a thing!